crypto/mem_sec.c

```diff
@@ -31,6 +31,10 @@
 # include <unistd.h>
 # include <sys/types.h>
 # include <sys/mman.h>
+# if defined(OPENSSL_SYS_LINUX)
```

Comment:
Looking again, the file is wrapped with defined(OPENSSL_SYS_LINUX) || defined(OPENSSL_SYS_UNIX), I'd prefer to keep checks like that.